### ISSUE TITLE
WireMock webhooks extension

### ISSUE THREAD

**Issue by tomakehurst** [viniciusffj/wiremock-http-requests-maker#1]
Hi,

Great to see new WM extensions being developed, but I thought it'd be remiss of me not to mention that I wrote a webhooks extension a while back (which I've not got around to publicising much): https://github.com/wiremock/wiremock-webhooks-extension

Maybe what you're doing goes beyond this, but I thought I'd point it out so that you don't needlessly replicate work already done.

Cheers!
Tom

**Comment by viniciusffj** [comment]
Hey @tomakehurst .

That's good know, I couldn't find anything so that's why I wrote my own extension.
I'll check it out and maybe send a PR if I need a new functionality.

Thanks.

**Comment by viniciusffj** [comment]
Hey @tomakehurst.

I just checked the webhooks and I'm think it will not work for my case. I really need to make the request before the response is served.

Any idea if something like this already exists?

**Comment by tomakehurst** [comment]
Right, I see. I'm not aware of any existing extension that does that.
